Lib/test/test_xml_etree.py

```diff
@@ -2320,6 +2320,7 @@ def __eq__(self, o):
         e.extend([ET.Element('bar')])
         self.assertRaises(ValueError, e.remove, X('baz'))
 
+    @unittest.skipIf(sys.gettrace(), "Skips under coverage.")
```

Comment:
Just for completeness: this would also skip it when using pdb then.. ;)

Reply:
For the record, I merely copied a line already used by another test. :)

Reply:
Looks like https://github.com/xdegaye/cpython/blob/dd9f66485d6bf8893ae0b02ff4d622ec343236c7/Lib/test/test_xml_etree.py#L1838 - but the reason might be different there (speed?).